data/human/human missions.txt

```diff
@@ -3631,18 +3631,27 @@ mission "Pirate Duel"
 	on enter
 		dialog `When you take off, you run a quick scan on all other ships that are taking off. You see no sign of the Esca. Maybe it's already waiting for you at Umbral.`
 	on enter "Umbral"
+		unmark "Umbral"
 		"reputation: Independent (Killable)" = -1000
 		conversation
+			branch "killed anglerfish"
+				has "anglerfish dead"
 			`When you enter Umbral, you continue to see no sign of the Esca. Suddenly, an alarm rings out as your sensors detect that a pirate Vanguard has entered the system alongside you. Before you can respond, your ship intercepts a hail, and it opens communication.`
 			`	The voice of the alleged captain of the Esca shouts out towards you. "Ho ho ho! Did you really think that I was the leader of the militia? Well, considering that you're here, I suppose you did. And now you'll pay for your mistake! Don't bother trying to call for help either, the militia don't patrol this system!"`
 			`	The Vanguard powers up its weapons.`
+				decline
+			label "killed anglerfish"
+			`Out of curiosity, you enter Umbral. As suspected, no sign of the Esca can be seen.`
 	
 	npc kill
 		government "Independent (Killable)"
 		personality heroic vindictive unconstrained launching
 			confusion 20
 		ship "Vanguard (Plasma Slow)" "Anglerfish"
 		dialog `The Anglerfish and the "captain of the Esca" have been destroyed. You consider going back to Wayfarer to see if you can grab some sort of reward for your effort.`
+		on kill
+			unmark "Umbral"
+			set "anglerfish dead"
```

Comment:
It might make sense to move this dialog into the `on kill` just for the sake of keeping things together. I'd also suggest adding an `on provoke` action that unmarks Umbral, sets your rep to negative (so that jumping to another system aside from Umbral doesn't unprovoke the ship), sets a condition that you attacked the Anglerfish early (that we can use in the `on kill` if we change the dialog to a conversation to branch if you kill the Anglerfish before you even got to Umbral, and to branch upon entering Umbral while you're already fighting the Anglerfish), and starts a dialog or conversation where the captain says something about you catching on to his act or something to that effect.